board/views.py

```diff
@@ -1,3 +1,74 @@
-from django.shortcuts import render
+import json
+
+from django.views   import View
+from django.http    import JsonResponse
+from django.db      import transaction
+
+from .models        import Review, ReviewImage, ReviewHostComment, ReviewHelpCount, FeedComment, FeedLikeCount, Question, QuestionComment
+
+class ReviewView(View):
+
+    def post(self, request):
+        data = json.loads(request.body)
+
+        try:
+            with transaction.atomic():
+                review = Review(
+                    order_id         = data['order_id'], 
+                    contents          = data['contents'],
+                    product_id       = data['product_id'],
+                    star_rating       = 0
+                    )
+
+                review.save()
+               
+                image_url_datas = data.get('url', None)
+                 
+                if image_url_datas:
+                    image_urls = [] 
+                
+                    for image_url in image_url_datas:
+                        image_urls.append(image_url)
+
+                        ReviewImage.objects.create(
+                        review_id = review.id,
+                        url       = image_url
+                        )
```

Comment:
line 16에서는 save() method로 생성을 했는데, line 33에서는 create()를 사용하신 이유가 있나요?
함수에 인자를 전달할 때 인덴트를 맞춰 가독성을 높여주시면 좋을 것 같아요~
```suggestion
                        ReviewImage.objects.create(
                                review_id = review.id,
                                url       = image_url
                        )
```

Reply:
위와 동일하게 save()로 수정하고, indent 맞춰 수정하였습니다.